WordleGame/WordleGame.xcodeproj/project.pbxproj

```diff
@@ -14,14 +14,19 @@
 		FA0E8C8F28189FD1000E9EB3 /* KeyboardManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA0E8C8E28189FD1000E9EB3 /* KeyboardManager.swift */; };
 		FA2D775D28141C2900D9600F /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA2D775C28141C2900D9600F /* AppDelegate.swift */; };
 		FA2D775F28141C2900D9600F /* SceneDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA2D775E28141C2900D9600F /* SceneDelegate.swift */; };
-		FA2D776128141C2900D9600F /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA2D776028141C2900D9600F /* ViewController.swift */; };
+		FA2D776128141C2900D9600F /* GameViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA2D776028141C2900D9600F /* GameViewController.swift */; };
 		FA2D776428141C2900D9600F /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = FA2D776228141C2900D9600F /* Main.storyboard */; };
 		FA2D776628141C2D00D9600F /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = FA2D776528141C2D00D9600F /* Assets.xcassets */; };
 		FA2D776928141C2D00D9600F /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = FA2D776728141C2D00D9600F /* LaunchScreen.storyboard */; };
 		FA2F69B2281704D000949BFC /* LetterEvaluation.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA2F69B1281704D000949BFC /* LetterEvaluation.swift */; };
+		FA45E843281D53600066C646 /* UIButton + Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA45E842281D53600066C646 /* UIButton + Extension.swift */; };
 		FA6652D628154D45004DE40A /* LetterBox.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA6652D528154D45004DE40A /* LetterBox.swift */; };
 		FA6652D82815561F004DE40A /* LetterBoxView.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA6652D72815561F004DE40A /* LetterBoxView.swift */; };
-		FA6652DA28155630004DE40A /* LetterBoxView.xib in Resources */ = {isa = PBXBuildFile; fileRef = FA6652D928155630004DE40A /* LetterBoxView.xib */; };
+		FAA3AD40282799E800F412D8 /* KeyboardButtonDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = FAA3AD3F282799E800F412D8 /* KeyboardButtonDelegate.swift */; };
+		FAAFE7CC282141FA0016A7B5 /* GameManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = FAAFE7CB282141FA0016A7B5 /* GameManager.swift */; };
+		FAAFE7CF282143950016A7B5 /* AllowedWords.txt in Resources */ = {isa = PBXBuildFile; fileRef = FAAFE7CE282143950016A7B5 /* AllowedWords.txt */; };
+		FAAFE7D128217B720016A7B5 /* LetterView.swift in Sources */ = {isa = PBXBuildFile; fileRef = FAAFE7D028217B720016A7B5 /* LetterView.swift */; };
+		FAAFE7D32821DD1D0016A7B5 /* UITextField + Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = FAAFE7D22821DD1D0016A7B5 /* UITextField + Extension.swift */; };
```

Comment:
Please, remove the deleted files from the project

<img width="255" alt="Screenshot 2022-05-11 at 22 26 51" src="https://user-images.githubusercontent.com/25828050/167930040-369ae496-f6d0-49f4-bcd3-08769dfd641c.png">

Reply:
Still no `AllowedWords` file. It's in the directory but not in a project

<img width="255" alt="Screenshot 2022-05-12 at 00 51 27" src="https://user-images.githubusercontent.com/25828050/167954147-452a52a0-6e58-4c8b-ab86-3186a3b1f3e9.png">